next-cloudinary/src/index.js

```diff
@@ -1,3 +1,7 @@
-export { default as CldImage } from './components/CldImage';
-export { cloudinaryLoader } from './loaders/cloudinary-loader';
-export { primary, text, position } from './constants/qualifiers';
\ No newline at end of file
+export { default as CldImage } from "./components/CldImage";
+export {
+  default as UploadWidget,
+  UploadButton,
```

Comment:
this need sto be `default as UploadButton` right?

Reply:
Only one default is allowed per module in index.js so I did the default as UploadButton there instead